components/fieldgroups/agentic/conversation-event.schema.json

```diff
@@ -0,0 +1,181 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/fieldgroups/agentic/conversationevent",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Conversation Event",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:tags": {
+    "industry": ["all"]
+  },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "description": "Conversation Event data including agentic information, prompt, response, feedback an associated metadata.",
+  "definitions": {
+    "conversationId": {
+      "properties": {
+        "xdm:id": {
```

Comment:
As per example file, you would need to rename `xdm:id` to `xdm:conversationID` (ID caps as per XDM naming convention)